PF1/assignment1/BinaryToOctal.java

```diff
@@ -0,0 +1,39 @@
+package assignment1;
+
+//convert binary number to octal format
+public class BinaryToOctal {
+	int convertBinaryToOctal(int n){
+		int no= n;
```

Comment:
Include line gap when you start any new block of code for code readability. Make similar changes in all assignment in future.